<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPainter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PaintWidget" name="paintWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Color"/>
   <addaction name="actionFill_Color"/>
   <addaction name="actionSelect_no_fill"/>
   <addaction name="actionSelect_full_fill"/>
   <addaction name="actionSelect_fill_with_border"/>
   <addaction name="actionsetGradient"/>
  </widget>
  <widget class="QToolBar" name="paintToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSwitch_to_Pen"/>
   <addaction name="actionSwitch_to_Brush"/>
   <addaction name="actionSwitch_to_Line"/>
   <addaction name="actionSwitch_to_Rect"/>
   <addaction name="actionSwitch_to_Circle"/>
   <addaction name="actionSwitch_to_Triangle"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/file_menu/resources/new_file.png</normaloff>:/file_menu/resources/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/file_menu/resources/open_file.png</normaloff>:/file_menu/resources/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/file_menu/resources/save_file.png</normaloff>:/file_menu/resources/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSwitch_to_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/line.png</normaloff>:/toolbar/resources/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Draw line</string>
   </property>
  </action>
  <action name="actionSwitch_to_Brush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/brush.png</normaloff>:/toolbar/resources/brush.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
   <property name="toolTip">
    <string>Switch to Brush</string>
   </property>
  </action>
  <action name="actionSwitch_to_Rect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/rectangle.png</normaloff>:/toolbar/resources/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Switch to Rectangle</string>
   </property>
  </action>
  <action name="actionSwitch_to_Circle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/circle.png</normaloff>:/toolbar/resources/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>Switch to Circle</string>
   </property>
  </action>
  <action name="actionSwitch_to_Triangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/triangle.png</normaloff>:/toolbar/resources/triangle.png</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
   <property name="toolTip">
    <string>Switch to Triangle</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/file_menu/resources/undo.png</normaloff>:/file_menu/resources/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/file_menu/resources/redo.png</normaloff>:/file_menu/resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSelect_Color">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/color_palette.png</normaloff>:/tool_properties/resources/color_palette.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="toolTip">
    <string>Select Line Color</string>
   </property>
  </action>
  <action name="actionFill_Color">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/color_area_fill.png</normaloff>:/tool_properties/resources/color_area_fill.png</iconset>
   </property>
   <property name="text">
    <string>Fill Color</string>
   </property>
   <property name="toolTip">
    <string>Select Fill Color</string>
   </property>
  </action>
  <action name="actionSelect_fill_with_border">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/fill_with_border.png</normaloff>:/tool_properties/resources/fill_with_border.png</iconset>
   </property>
   <property name="text">
    <string>Fill with border</string>
   </property>
   <property name="toolTip">
    <string>Select fill with border</string>
   </property>
  </action>
  <action name="actionsetGradient">
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/gradient.png</normaloff>:/tool_properties/resources/gradient.png</iconset>
   </property>
   <property name="text">
    <string>Gradient</string>
   </property>
   <property name="toolTip">
    <string>Set Gradient Color</string>
   </property>
  </action>
  <action name="actionSelect_full_fill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/full_fill.png</normaloff>:/tool_properties/resources/full_fill.png</iconset>
   </property>
   <property name="text">
    <string>Full fill</string>
   </property>
   <property name="toolTip">
    <string>Select Full Fill</string>
   </property>
  </action>
  <action name="actionSelect_no_fill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/tool_properties/resources/no_fill.png</normaloff>:/tool_properties/resources/no_fill.png</iconset>
   </property>
   <property name="text">
    <string>No Fill</string>
   </property>
   <property name="toolTip">
    <string>Select no fill</string>
   </property>
  </action>
  <action name="actionSwitch_to_Pen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="menu_icons.qrc">
     <normaloff>:/toolbar/resources/pen.png</normaloff>:/toolbar/resources/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="toolTip">
    <string>Switch to Pen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header>paintwidget.h</header>
   <container>1</container>
   <slots>
    <slot>switchTool()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="menu_icons.qrc"/>
 </resources>
 <connections/>
</ui>
